js/templates/modals/listingDetail/listing.html

```diff
@@ -1,11 +1,187 @@
+          <% } else { %>
+            style="background-image: url('../imgs/defaultListing.png')"
+          <% } %>
+          ></div>
+          <div class="pad txCtr">
+            <a class="tx5 js-gotoPhotos">
+              <u><%= ob.polyT('listingDetail.viewPhotos', { count: ob.item.images.length }) %></u>
+            </a>
+          </div>
+        </div>
+        <div class="flexExpand">
+          <div class="buyBox clrP clrBr">
+            <div class="flexColRows flexHCent gutterV">
+              <!-- this will loop through each variant, just placeholder code for formatting-->
+              <!-- commented out until variants exist
+              <div class="flexVCent gutterHLg">
+                <div class="h5 txUnl">Size</div>
+                <div class="flexExpand">
+                  <select class="js-variantSelect" name="">
+                    <option>some option 1</option>
+                    <option>some option 2</option>
+                    <option>some option 3</option>
+                  </select>
+                </div>
+              </div>
+              -->
+              <button class="btnHg clrBAttGrad clrBr3 clrTOnEmph">Buy Now</button>
+              <div class="flexHCent gutterH">
+                <!--<div class="ion-star"> X.X</div>--><!-- placeholder until reviews exist -->
+                <!--<div class="ion-chatbubble"> X.X</div>--><!-- placeholder until reviews exist -->
+                <% if(ob.shipsFreeToMe) { %>
+                <div class="txCtr">
+                  <a class="clrE1 clrTOnEmph phraseBox txNoUnd js-freeShippingLabel"><%= ob.polyT('listingDetail.freeShippingBanner') %></a>
+                </div>
+                <% } %>
+              </div>
+            </div>
+          </div>
+          <div class="flexHCent gutterHLg tx5 rowLg">
+            <div>
+              <%= ob.polyT('listingDetail.type', { type: `<b>${ob.polyT(`formats.${ob.metadata.contractType}`)}</b>` }) %>
+            </div>
+            <% if (ob.item.condition) { %>
+              <div><%= ob.polyT('listingDetail.condition', { condition: `<b>${ob.polyT(`conditionTypes.${ob.item.condition}`)}</b>` }) %></div>
+            <% } %>
+          </div>
+          <% if(ob.item.tags.length) { %>
+            <hr class="rowLg">
+            <h5><%= ob.polyT('listingDetail.tags') %></h5>
+            <div class="tagWrapper">
+              <% ob.item.tags.forEach(function(tag){
+              print(`<div class="btn tag clrBr">${tag}</div>`);
+              }); %>
+            </div>
+          <% } %>
+        </div>
+      </div>
+    </div>
+
+    <div class="contentBox padLg clrP clrBr clrSh3">
+      <h2 class="txUnb"><%= ob.polyT('listingDetail.description') %></h2>
+      <%= ob.item.description %>
+    </div>
+
+  <div class="flexRow gutterH" id="avatarCropper">
```

Comment:
I think this is the block @drwasho pointed out that shouldn't be here. Looks like a missed copy-paste cleanup.